hail_scripts/v02/update_reference_data_in_existing_index.py

```diff
@@ -0,0 +1,142 @@
+        mt = CLINVARSchema(mt, clinvar_data=clinvar).clinvar()
+
+    if args.update_hgmd:
+        hgmd = hl.read_table(hgmd_ht_path)
+        mt = HGMDSchema(mt, hgmd_data=hgmd).hgmd()
+
+    if args.update_cidr:
+        cidr = hl.read_table(cidr_ht_path)
+        mt = CIDRSchema(mt, cidr_data=cidr).cidr()
+
+    mt = mt.aIndex().doc_id()
+    mt = mt.select_annotated_mt()
+
+    variant_count = mt.count_rows()
+    logger.info("\n==> exporting {} variants to elasticsearch:".format(variant_count))
+
+    row_table = mt.rows().flatten()
+    row_table = row_table.drop(row_table.locus, row_table.alleles)
+
+    elasticsearch_client.export_table_to_elasticsearch(
+        row_table,
+        index_name=index_name,
+        index_type_name="variant",
+        block_size=args.block_size,
+        elasticsearch_write_operation=ELASTICSEARCH_UPDATE,
+        elasticsearch_mapping_id="docId",
+        verbose=False,
+        delete_index_before_exporting=False,
+        ignore_elasticsearch_write_errors=False,
+        export_globals_to_index_meta=True,
+    )
+
+
+p = argparse.ArgumentParser()
+p.add_argument("--host", help="Elasticsearch host", default=os.environ.get("ELASTICSEARCH_SERVICE_HOSTNAME"))
+p.add_argument("--port", help="Elasticsearch port", default="9200")
+p.add_argument("--block-size", help="Block size to use when exporting to elasticsearch", default=1000, type=int)
+
+#p.add_argument("--download-latest-clinvar-vcf", action="store_true", help="First download the latest GRCh37 and GRCh38 clinvar VCFs from NCBI.")
+p.add_argument("--update-clinvar", action="store_true", help="Update clinvar fields.")
+p.add_argument("--update-hgmd", action="store_true", help="Update hgmd fields.")
+p.add_argument("--update-cidr", action="store_true", help="Update cidr fields.")
+
+# these datasets don't get new versions as frequently as ClinVar or HGMD but they do occasionally
+#p.add_argument("--update-primate-ai", action="store_true", help="Update PrimateAI fields.")
+#p.add_argument("--update-splice-ai", action="store_true", help="Update SpliceAI fields.")
+
+p.add_argument("--index-name", help="Elasticsearch index name. If specified, only this index will be updated.")
+p.add_argument("--dataset-path", help="(optional) Path of variant callset. If not specified, the original "
+    "vcf/vds path from which the data was loaded will be used.")
+p.add_argument("--genome-version", help="Genome build: 37 or 38", choices=["37", "38"])
+
+p.add_argument("--all", help="Update all elasticsearch indices. This option is mutually-exclusive "
+    "with --index-name, --dataset-path, and --genome-version.", action="store_true")
+
+
+args = p.parse_args()
+
+#if args.download_latest_clinvar_vcf:
+#    for genome_version in ["37", "38"]:
+#        vds = download_and_import_latest_clinvar_vcf(hc, genome_version)
+#        write_vds(vds, CLINVAR_VDS_PATH.format(genome_version=genome_version))
+
+if args.index_name and not args.all:
```

Comment:
If I'm reading this correct, the order of precedence is all and then the individual index name. If so, this is equivalent to

```
if args.all:
  ...
elif args.index_name:
  ...
else:
  ...
```